lib/task/import/validate/csvValidatorResult.class.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * CSV validation result class.
+ *
+ * @author     Steve Breker <sbreker@artefactual.com>
+ */
+class CsvValidatorResult
+{
+    // Integer type to allow comparison of severity values.
+    const RESULT_INFO = 0;
+    const RESULT_WARN = 1;
+    const RESULT_ERROR = 2;
+
+    const TEST_TITLE = 'title';
+    const TEST_STATUS = 'status';
+    const TEST_RESULTS = 'results';
+    const TEST_DETAIL = 'details';
```

Comment:
I think this should be `TEST_DETAILS` (plural) to match `TEST_RESULTS`.

Reply:
👍

Reply:
Change completed.